registry/time/helpers_test.go

```diff
@@ -0,0 +1,60 @@
+package time_test
```

Comment:
On helpers_test you can be directly on the time package to test the private method ✅